plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftMetastoreUtil.java

```diff
@@ -146,10 +147,10 @@
 {
     private static final String PUBLIC_ROLE_NAME = "public";
     private static final String ADMIN_ROLE_NAME = "admin";
-    private static final String NUM_FILES = "numFiles";
+    public static final String NUM_FILES = "numFiles";
```

Comment:
public fields above private. 

I would just copy fields and keep them private here